pyproject.toml

```diff
@@ -0,0 +1,118 @@
+[project]
+authors = [
+    {name = "Webkom", email = "webkom@abakus.no"},
+]
+dependencies = [
+    "django==4.0.8",
+    "ipython==8.7.0",
+    "psycopg2-binary==2.9.5",
+    "unicode-slugify==0.1.5",
+    "pyyaml==6.0",
+    "redis==4.4.0",
+    "hiredis==2.1.0",
+    "certifi==2022.12.7",
+    "elasticsearch==7.16.2",
+    "celery==5.2.7",
+    "stripe==3.4.0",
+    "boto3==1.26.31",
+    "beautifulsoup4==4.11.1",
+    "bleach==5.0.1",
+    "requests==2.28.1",
+    "ldap3==2.9.1",
+    "google-api-python-client==2.70.0",
+    "oauth2client>=4.1.3",
+    "premailer==3.10.0",
+    "libthumbor==2.0.2",
+    "django-health-check==3.17.0",
+    "djangorestframework==3.14.0",
+    "drf-jwt==1.19.2",
+    "djangorestframework-camel-case==1.3.0",
+    "django-extensions==3.2.1",
+    "django-autoslug==1.9.8",
+    "django-oauth-toolkit==1.6.3",
+    "django-cors-headers==3.13.0",
+    "django-mptt==0.14.0",
+    "django-filter==22.1",
+    "django-environ==0.9.0",
+    "django-redis==5.2.0",
+    "django-ipware==4.0.2",
+    "django-phonenumber-field[phonenumberslite]==7.0.1",
+    "django-push-notifications==3.0.0",
+    "django-ical==1.8.3",
+    "icalendar==5.0.3",
+    # Channels
+    "channels==3.0.5",
+    "channels-redis==4.0.0",
+    "daphne==3.0.2",
+    "flatbuffers==22.12.6",
+    # Logging/errors
+    "structlog==21.5.0",
+    "structlog-sentry==1.4.0",
+    "sentry-sdk==1.12.0",
+    "analytics-python==1.2.9",
+    # Docs
+    "Pygments==2.13.0",
+    "Markdown==3.4.1",
+    "coreapi==2.3.3",
+    "aiosmtpd==1.4.2",
+]
+requires-python = ">=3.11"
+license = {text = "MIT"}
+
+[project.urls]
+Homepage = "https://lego.abakus.no"
+Repository = "https://github.com/webkom/lego"
+Documentation = "https://lego.abakus.no/docs/"
+ 
+[tool.pdm.scripts]
+start = "python manage.py runserver"
```

Comment:
What do people think of this? I kind of like the recemplance of npm/yarn

Reply:
I kinda don't like it tbh. I think it hides what's going on too much. It's a bit weird since in the django/python world, django and `manage.py` by extension is sort of the script manager, and hiding that behind pdm is unnecessary and strange.

Also, I would like to vouch for the Makefile. It's just more universal, easier to read and edit and faster. I also kind of like having it the other way round, where you have Makefiles as a wrapper around everything else that _needs_ to be handled by pdm, npm or whatever else. (this article sums it up pretty well: https://www.yieldcode.blog/post/why-you-should-adpot-makefile-in-all-of-your-projects)

Reply:
I agree that we should use `manage.py`.

I don't see why we need a Makefile when we have this integrated? make commands are _much_ slower. E.g. try to compare `pdm format` and `make fixme`. fixme requires docker images to run and installs everything without using the cache. So, faster, no. Easier to edit? They are almost never edited and how is this any more difficult?